pages/index.html

```diff
@@ -4,9 +4,31 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>HOME</title>
+  <link href="../stylesheets/style.css" rel="stylesheet">
+  <link href="../stylesheets/index.css" rel="stylesheet">
 </head>
+
 <body>
   
+  <header>
+    <h1>DOMINIQUE DABA</h1>
+  </header>
+
+  <section>
+    
+    <nav>
+      <a href="about.html">ABOUT</a>
+      <a href="portfolio.html">PORTFOLIO</a>
+    </nav>
+  
+    <img alt="Orange tree with fruit against an orange, textured wall." src="../images/orange_tree.jpg">
+  
+  </section>
+  
+  <footer>
```

Comment:
Make sure you put content inside your footer tag! It's good to have a copyright notice in your footer if nothing else.